docs/metrics.asciidoc

```diff
@@ -60,3 +60,69 @@ This value is normalized by the number of CPU cores and it ranges from 0 to 100%
 
 The Resident Set Size,
 the amount of memory the process occupies in main memory (RAM).
+
+[float]
+[[metric-nodejs.active_handles]]
+=== `nodejs.active_handles`
+
+* *Type:* Long
+* *Format:* Counter
+
+The number of active libuv handles,
+likely held open by currently running I/O operations.
+
+[float]
+[[metric-nodejs.active_requests]]
+=== `nodejs.active_requests`
+
+* *Type:* Long
+* *Format:* Counter
+
+The number of active libuv requests,
+likely waiting for a response to an I/O operation.
+
+[float]
+[[metric-system.process.cpu.user.ticks]]
+=== `system.process.cpu.user.ticks`
+
+* *Type:* Long
+* *Format:* Counter
+
+The number of CPU cycles spent executing application code.
+
+[float]
+[[metric-system.process.cpu.system.ticks]]
+=== `system.process.cpu.system.ticks`
+
+* *Type:* Long
+* *Format:* Counter
+
+The number of CPU cycles spent executing kernel code as a result of application activity.
+
+[float]
+[[metric-nodejs.eventloop.delay.ns]]
+=== `nodejs.eventloop.delay.ns`
+
+* *Type:* Float
+* *Format:* Nanoseconds
+
+The number of nanoseconds of event loop delay.
+Event loop delay is sampled every 10 milliseconds so smaller delays may not appear if they do not impact overall timer execution.
```

Comment:
Say you set a timer to fire in 10ms. But 5ms later some piece of work kicks off that blocks the event loop for 5ms. This will not be discovered because the timer is still able to fire right after the blocking operation finishes.

But if the blocking operation had lasted 6ms, you'd have seen that the timer was 1ms delayed.

Reply:
Ah, of course! Thanks :)

Maybe I was just being a bit derpy and this is immediately clear to everyone else, but I think it'd be a little clearer if described in terms of the sampling period rather than the timer. e.g.

> Event loop delay is sampled every 10 milliseconds. Delays shorter than 10ms may not be observed, for example if a blocking operation starts and ends within the same sampling period.

YMMV, feel free to ignore.

Reply:
I like your description much better actually. Thanks for helping us improve the docs 😊

Reply:
fixed